tests/manage/z_cluster/test_log_rotate.py

```diff
@@ -0,0 +1,190 @@
+import logging
+import time
+import pytest
+import re
+
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources.storage_cluster import verify_storage_cluster
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.ocs.cluster import ceph_health_check
+from ocs_ci.ocs.resources import pod
+from ocs_ci.ocs.exceptions import TimeoutExpiredError
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    ManageTest,
+    tier2,
+    bugzilla,
+    skipif_ocs_version,
+    skipif_external_mode,
+    ignore_leftovers,
+    config,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier2
+@ignore_leftovers
+@bugzilla("2116416")
+@skipif_external_mode
+@skipif_ocs_version("<4.10")
+@pytest.mark.polarion_id("OCS-4684")
+class TestLogsRotate(ManageTest):
+    """
+    Test Logs Rotate
+
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request):
+        def finalizer():
+            log.info("Delete logCollector from storage cluster yaml file")
+            storagecluster_obj = OCP(
+                resource_name=constants.DEFAULT_CLUSTERNAME,
+                namespace=defaults.ROOK_CLUSTER_NAMESPACE,
+                kind=constants.STORAGECLUSTER,
+            )
+            params = '[{"op": "remove", "path": "/spec/logCollector"}]'
```

Comment:
maybe we should first check if this entry is present and then remove it if it is found.

Reply:
The system checks the log size every 15 min